src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/planner/QueryPlannerTest.java

```diff
@@ -78,12 +77,12 @@
     @Mock
     protected Client client;
 
-    @Spy
```

Comment:
I guess Spy object was in use for some reason. So nothing broken after switch to Mock?

Reply:
Yes, all tests passed.

Reply:
I see. Interesting.